text/0005-application-formalization.md

```diff
@@ -0,0 +1,191 @@
+- Start Date: 10/09/2023
+- Contributors:
+  - [Craig McNally](cmcnally@ebsco.com)
+  - [Vince Bareau](vbareau@ebsco.com)
+- RFC PRs:
+  - DRAFT REVIEW: https://github.com/folio-org/rfcs/pull/14
+  - PRELIMINARY REVIEW: TBD
+  - PUBLIC REVIEW: TBD
+  - FINAL REVIEW: TBD
+- Outcome: <!-- Leave this blank. Will eventually be either ACCEPTED or REJECTED -->
+
+# Application Formalization
+
+## Summary
+It is proposed that an Application be defined as the minimal but complete set of elements which together are intended to deliver a specific solution to Folio. The intention is to group together all the closely related components which will need to evolve together, but independently of other parts (i.e. other applications) of Folio.
+
+## Motivation
+The existing Folio project lacks a formal definition of an Application structure. A consequence is that Folio is stuck in a mode of monolithic releases which are ever increasing in size.  There have also been repeated complaints that the setup and management of Folio can be difficult due to the number modules involved.
+
+## Scope
+* Application composition
+* Application versioning
+* Application dependencies
+* Application definition/descriptors
+* Application management is ***out of scope***
+* Application-specific UI bundles is ***out of scope***
+* How movement of modules between Applications will work is ***out of scope***
+
+## Detailed Explanation/Design
+
+### Terminology
+* Application:  The minimal but complete set of elements which together are intended to deliver a specific solution to Folio.
+* Bounded context:  The boundary of the microservice.  It "owns" the storage for the service.
```

Comment:
What does service refer to in the context of this document?

Is it a module, an application or something else?